keyboards/mino_plus/hotswap/info.json

```diff
@@ -0,0 +1,473 @@
+{
```

Comment:
Please run `qmk format-json` on this file. Make sure the branch is up to date with upstream first.

Reply:
@fauxpark I just updated the file. If there are any issue let me know.